src/main/java/com/GG/T9AgeCombat/entities/EquipmentDTO.java

```diff
@@ -0,0 +1,40 @@
+package com.GG.T9AgeCombat.entities;
+
+import com.GG.T9AgeCombat.enums.EquipmentEnum;
+import com.GG.T9AgeCombat.enums.EquipmentTypeEnum;
+import com.GG.T9AgeCombat.models.EquipmentClassification;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import lombok.Value;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.io.Serializable;
+import java.util.List;
+
+@Value
+@Builder
+@Entity
+@NoArgsConstructor(force = true)
+@AllArgsConstructor
+public class EquipmentDTO implements Serializable {
+    @Id
+    long id;
+    String name;
+    String equipmentType;
+    @Id
+    String specialRuleName;
+    int specialRuleValue;
+    String modificationValue;
+    String limitationValue;
+    String timingValue;
```

Comment:
With this setup is a unit going to have multiple instances of an equipment for each special rule on that equipment? Would we rather each equipment have a list of SpecialRules like the Unit does?

Reply:
No they won't. See Equipment.java for the model that actually goes on the Unit.   I'll move the builder method to an equipment service class though.